packages/integration_test/lib/integration_test_driver.dart

```diff
@@ -64,7 +64,7 @@ Future<void> writeResponseData(
 ///
 /// ```
 Future<void> integrationDriver({
-  Duration timeout = const Duration(minutes: 1),
+  Duration timeout = const Duration(minutes: 20),
```

Comment:
I can't kill this because it's part of public API. If we want we could deprecate it. At any rate I'm raising it to something that should be closer to our overall test timeout.

Reply:
/cc @jiahaog in case this impacts internal infra somehow.

Reply:
Nope, this is fine 🙂